src/components/utils/src/file_system.cc

```diff
@@ -73,15 +73,8 @@ size_t file_system::DirectorySize(const std::string& path) {
   size_t size = 0;
```

Comment:
There shouldn't be code in this PR that is not related to RC. Are these changes necessary for RC to work properly?